pkg/trivy/model.go

```diff
@@ -24,6 +24,11 @@ type VersionInfo struct {
 	VulnerabilityDB *Metadata `json:"VulnerabilityDB"`
 }
 
+type Layer struct {
+	Digest string `json:"Digest"`
```

Comment:
nit: JSON tag is unnecessary, right?

Reply:
Yes but I always keep it even if it's redundant. For a day, when someone comes and accidentally or for good reason refactors the name of the `Digest` property and we break the JSON output.

Reply:
I got it